src/states/infinitepeps.jl

```diff
@@ -134,6 +134,10 @@ Base.:/(ψ::InfinitePEPS, α::Number) = InfinitePEPS(ψ.A / α)
 LinearAlgebra.dot(ψ₁::InfinitePEPS, ψ₂::InfinitePEPS) = dot(ψ₁.A, ψ₂.A)
 LinearAlgebra.norm(ψ::InfinitePEPS) = norm(ψ.A)
 
+## not a nice solution, but it works
+Base.:+(ipeps::InfinitePEPS, x::NamedTuple) = InfinitePEPS(ipeps.A .+ x.A)
+Base.:+(x::NamedTuple, ipeps::InfinitePEPS) = InfinitePEPS(ipeps.A .+ x.A)
```

Comment:
Is this something that is required because of Zygote? I think if you are a bit careful with also defining rrules for constructors and `getproperty`, you can avoid the case where it generates weird tangent objects.

Reply:
You are correct. This implementation is unsatisfactory for me. If you have any ideas on how to improve this, please provide an implementation. Thank you.